components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,84 @@
+const reddit = require("../../reddit.app.js");
+const get = require("lodash.get");
+
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
+  description: "Emmits a new link submitted to a subreddit",
+  version: "0.0.2",
```

Comment:
This version only needs to be incremented each time the event source is published to production (this happens when you merge to `master`). Can you change back to `0.0.1`?

Reply:
Oh, interesting. Glad to know. Do that match somehow to the version number from semantic versioning mentioned in the pd docs? (in this case the version number being changed back is a patch version number)

Reply:
Technically yes, we should be implementing semver here, although we don't use it in the product (sources aren't automatically updated when new versions are shipped).